packages/nimble-components/src/select/index.ts

```diff
@@ -510,12 +515,19 @@ export class Select
         }
 
         if (!this.options?.includes(focusTarget as ListboxOption)) {
+            let currentActiveIndex = this.openActiveIndex ?? this.selectedIndex;
             this.open = false;
-            if (this.selectedIndex === -1) {
-                this.selectedIndex = this.indexWhenOpened!;
+            if (currentActiveIndex === -1) {
+                currentActiveIndex = this.selectedIndex;
+                if (this.selectedIndex >= 0) {
+                    (
+                        this.options[this.selectedIndex]! as ListOption
+                    ).activeOption = true;
```

Comment:
What's the purpose of setting `activeOption` to `true` here? Didn't the select just close? I would expect `activeOption` to be updated when the select opens again, but I wouldn't expect that attribute to need to stay up-to-date on options while the select is closed.

Reply:
Yeah, this is not needed.